src/utils/style-formatter.js

```diff
@@ -29,6 +29,16 @@ const formatColorProperty = (path, inputColor, defaultColor) => {
   return `${path}: ${color === 'none' ? defaultColor : color};`;
 };
 
+const checkDisabled = (style) => {
+  if (style.useEnabledCondition) {
+    const EC = style.enabledCondition;
+    if (EC === 0) {
+      return true;
+    }
+  }
+  return false;
+};
```

Comment:
This can be written as `return style.useEnabledCondition && style.enabledCondition === 0`